src/components/HomeContainer/SuccessStories/SuccessStories.module.css

```diff
@@ -78,7 +79,6 @@
 @media (min-width: 960px) {
   .wrapper {
     padding-block: 140px;
-    max-width: 1148px;
```

Comment:
This is the only edit to what @joanwolk added in the previous PR. It includes the 165px L/R padding so the content width is quite narrow for desktop.

Reply:
Ignore my previous comment. In my latest commit, I've added max-width back but made the box-sizing to content-box.